### ISSUE TITLE
Host marvin-mk2

### ISSUE THREAD

**Issue by timokau** [nix-community/infra#24]
I'd like to [move](https://github.com/NixOS/rfcs/pull/30#issuecomment-640126206) the [marvin-mk2](https://discourse.nixos.org/t/rfc-marvin-mk2-the-pull-request-bot/7547) bot ([repo](https://github.com/timokau/marvin-mk2)) onto nix-community infrastructure before starting to test it on nixpkgs.

It should consume very little resources. Would that be possible? If so, how should I go about it?

**Comment by adisbladis** [comment]
Fixed! We're nov hosting it :)

**Comment by timokau** [comment]
> Also, all the current people who have access to the secrets will need to be okay with you having access.

Should I open an issue about this? Having to nag one of you every time I'd like to deploy an update is not ideal, especially since I can't control when a deploy happens :)

**Comment by ryantm** [comment]
Yes, it's probably best to open another issue, then I can ping all the people with access to get their okay.
